### DISCUSSION TITLE
Update 7.4

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#399] [Release]
## What's Changed
* Bump androidx.annotation:annotation from 1.5.0 to 1.6.0 by @dependabot in https://github.com/TBog/TBLauncher/pull/380
* Bump lifecycle_version from 2.5.1 to 2.6.1 by @dependabot in https://github.com/TBog/TBLauncher/pull/385
* Bump com.android.tools:desugar_jdk_libs from 1.1.8 to 2.0.3 by @dependabot in https://github.com/TBog/TBLauncher/pull/387
* Bump androidx.core:core-ktx from 1.9.0 to 1.10.0 by @dependabot in https://github.com/TBog/TBLauncher/pull/388
* Bump org.jetbrains.kotlin:kotlin-gradle-plugin from 1.7.22 to 1.8.21 by @dependabot in https://github.com/TBog/TBLauncher/pull/390
* Bump com.android.tools.build:gradle from 7.2.2 to 8.0.1 by @dependabot in https://github.com/TBog/TBLauncher/pull/392
* Bump com.google.android.material:material from 1.8.0 to 1.9.0 by @dependabot in https://github.com/TBog/TBLauncher/pull/393
* Update CI to use java 17 by @TBog in https://github.com/TBog/TBLauncher/pull/396
* Fix for the Interference between the search bar and the widget desktop. by @lucasnasc2 in https://github.com/TBog/TBLauncher/pull/381
* Bump com.android.tools.build:gradle from 8.1.0 to 8.1.1 by @dependabot in https://github.com/TBog/TBLauncher/pull/398
* Translations update from Hosted Weblate by @weblate in https://github.com/TBog/TBLauncher/pull/373

## New Contributors
* @lucasnasc2 made their first contribution in https://github.com/TBog/TBLauncher/pull/381

**Full Changelog**: https://github.com/TBog/TBLauncher/compare/v7.3...v7.4

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v7.4'>Update 7.4</a>.</em>

**Comment by ildar** [comment]
@TBog, thanks a lot for your ongoing work on this launcher.
Unfortunately I missed (been busy) the release where the contact icon tap
behavior changed to "Call". I still think that the standard dialer behavior
fits better here: open the contact card.
I couldn't find the option for this. Am I missing it?

**Comment by TBog** [comment]
In your case long touch on the call icon of any contact and select "Reset default action"

**Comment by ildar** [comment]
Ah! it helped, thanks a lot!
